queue/queue.go

```diff
@@ -0,0 +1,32 @@
+package party
+
+import (
+)
+
+type Queue struct {
+	 songList []string
+}
+
+func NewQueue() *Queue {
+	q := Queue{
+		songList:make([]string, 10),

```

Comment:
I think what we really want is a set. Even though this is a song queue, it's not actually a queue.